### DISCUSSION TITLE
Which backend?

### DISCUSSION THREAD

**Discussion by DavidBDiligence** [Atome-FE/llama-node#82] [Q&A] [answered]
Hi, I'm trying to get started and the instructions call for the installation of one of three inference backends (llama.cpp, llm, or rwkv.cpp) with no guidance about how to choose one.

Would you please provide a quick-start recommendation for a beginner?

**Comment by elijah629** [comment, marked as answer]
Personally I would recommend llama.cpp, it works well with many different types of models
